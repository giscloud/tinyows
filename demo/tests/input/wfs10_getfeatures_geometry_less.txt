service=WFS&version=1.0.0&request=GetFeature&typename=tows:geometry_less
