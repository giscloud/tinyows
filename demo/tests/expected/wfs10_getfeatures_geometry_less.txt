Content-Type: text/xml; subtype=gml/2.1.2

<?xml version='1.0' encoding='UTF-8'?>
<wfs:FeatureCollection
 xmlns:tows='http://www.tinyows.org/'
 xmlns:wfs='http://www.opengis.net/wfs'
 xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
 xmlns:gml='http://www.opengis.net/gml'
 xmlns:xsd='http://www.w3.org/2001/XMLSchema'
 xmlns:ogc='http://www.opengis.net/ogc'
 xmlns:xlink='http://www.w3.org/1999/xlink'
 xmlns:ows='http://www.opengis.net/ows'
 xsi:schemaLocation='http://www.tinyows.org/
    http://127.0.0.1/cgi-bin/tinyows?service=WFS&amp;version=1.0.0&amp;request=DescribeFeatureType&amp;Typename=tows:geometry_less   http://www.opengis.net/wfs
   http://schemas.opengis.net/wfs/1.0.0/WFS-basic.xsd
   http://www.opengis.net/gml
   http://schemas.opengis.net/gml/2.1.2/feature.xsd'
>
<gml:boundedBy><gml:null>missing</gml:null></gml:boundedBy>
  <gml:featureMember>
   <tows:geometry_less fid="geometry_less.1">
   <tows:intcol>123</tows:intcol>
   <tows:textcol>foo</tows:textcol>
   </tows:geometry_less>
  </gml:featureMember>
</wfs:FeatureCollection>
